_data/wai-evaluation-tools-list/submissions/cde07110-0a38-11f0-a876-557e596872ce.json

```diff
@@ -0,0 +1,61 @@
+{
+  "a11yloc": "",
+  "actrules": "",
+  "assists": [
+    "Generating reports of evaluation results",
+    "Displaying information within web pages"
+  ],
+  "automated": [
+    "Single page/screen",
+    "Sample of pages/screens",
+    "Entire website / software / product"
```

Comment:
I can't find evidence that the plugin allows to test multiple pages/screens in a single evaluation.

Reply:
The extension only tests the page open in the current tab.